rosetta-source/src/main/rosetta/legaldocumentation-common-type.rosetta

```diff
@@ -153,18 +159,27 @@ type AgreementName: <"Specifies the agreement name through an agreement type and
         if creditSupportAgreementMarginType exists
         then creditSupportAgreementType exists
 
-type UmbrellaAgreement: <"A class to specify a set of legal entities which are part of a legal agreement beyond the two contracting parties to that agreement. This data representation reflects the ISDA Create representation.">
+type UmbrellaAgreement: <"Specifies a set of legal entities which are part of a legal agreement beyond the two contracting parties to that agreement.">
 
-    isApplicable boolean (1..1) <"The determination of whether Umbrella Agreement terms are Applicable (True), or Not Applicable (False)">
-    language string (0..1) <"The language associated with the umbrella agreement, and which applies to all the parties to the umbrella agreement.">
-    parties UmbrellaAgreementEntity (0..*) <"Underlying principals to the umbrella agreement.">
+    agreementSet UmbrellaAgreementSet (1..*) <"The language associated with the umbrella agreement, and which applies to all the parties to the umbrella agreement.">
+    parties UmbrellaAgreementEntity (2..*) <"Specifies the underlying entities to the agreement that either act as legal parties to the agreement or represent Legal constructs that are managed by a party to the agreement but do not represent a Legal Entity as of themselves.">
```

Comment:
Aware the `Legal Entity` capitalisation is standard, but gave me pause at `Legal constructs`. Very minor